index.html

```diff
@@ -96,30 +96,39 @@
 
       function checkForStandardMfgs(firstThree) {
         if(-1 == knownMacs.indexOf(firstThree.join(""))){
-          displayWarning(
+          displayError(
             "The MAC address does not appear to be from a PittMesh standard manufacturer (" +
             standardMfgs.join(", ") +
             "). The IP address calculation may be invalid.");
         }
       }
 
-      function log(text) {
+      function log(text, style = "") {
         var date = new Date();
         var timestamp = [date.getHours(), date.getMinutes(), date.getSeconds()].join(":");
         var message = "[" + timestamp + "] " + text;
 
-        transcriptEl.appendFirst(createLogLineElement(message));
-        console.log(message);
+        transcriptEl.appendFirst(createLogLineElement(message, style));
+        console.log("%c"+message, style);
+      }
+
+      function displayError(text) {
+        log(text, 'color:red; font-family:helvetica,sans-serif; font-weight:bold; text-shadow:0 1px 2px rgba(0,0,0,.5);');
```

Comment:
For the html created from createLogLineElement() function, yes, but not for the styling of console.log (at least that I am familiar with). See http://stackoverflow.com/questions/24828107/javascript-adding-style-to-the-text-of-console-log